### PR TITLE
Added error values to column, beautyfied py code, and Client Side Plotting

### PR DESCRIPTION
**Pull request by ricardovb**
Now columns supports add values with a related error value.  All model package, dave engine, py tests and Html files were involved in this change.

### CODE REVIEW

**Comment by ricardovb** [comment]
Great, all Python Project has been beautyfied!! 💃

File: src/main/resources/static/scripts/outputpanel.js [hunk 1]
```diff
@@ -50,25 +46,20 @@ function OutputPanel (classSelector, toolBarSelector, service, onFiltersChangedF
...
+                            { table:"txt_table", column:"Amplitude" } ]
                   },
                   this.service,
                   this.onFiltersChangedFromPlot,
                   this.$toolBar
                 ),
 
                 new Plot(
-                  "Time_Rate_Amplitude_" + filename,
+                  "Time_Frecuency_" + filename,
```
Comment (pbalm) [inline src/main/resources/static/scripts/outputpanel.js:57]: Typo: frequency

File: src/main/resources/static/scripts/plot.js [hunk 2]
```diff
@@ -51,34 +54,58 @@ function Plot(id, plotConfig, service, onFiltersChangedFn, toolbar) {
...
+                                    currentObj.plotConfig.styles.labels[1],
+                                    data[3].values);
+
+   } else if (currentObj.plotConfig.styles.type == "scatter") {
+      plotlyConfig = get_plotdiv_scatter(data[0].values, data[1].values,
+                                        data[2].values,
+                                        currentObj.plotConfig.styles.labels[0],
+                                        currentObj.plotConfig.styles.labels[1],
+                                        'Amplitude<br>Map');
+   }
```
Comment (pbalm) [inline src/main/resources/static/scripts/plot.js:85]: Add an error: else log.error("unknown plot type: " + currentObj.plotConfig.styles.type")

**Review by pbalm — APPROVED** [review]
Awesome. I was sure you hadn't updated the tests... But you did. Really good.

**Comment by pbalm** [comment]
I'm sorry that due to the other PR I've created some conflicts in this PR!
Now you have to merge the master from this repo onto your own branch. We can do it tomorrow in the bar if you want.

**Pull request merged**